docs/reference/ml/ml-shared.asciidoc

```diff
@@ -863,9 +863,10 @@ example, `1575402236000 `.
 end::model-snapshot-id[]
 
 tag::model-snapshot-retention-days[]
-The time in days that model snapshots are retained for the job. Older snapshots
-are deleted. The default value is `1`, which means snapshots are retained for
-one day (twenty-four hours).
+Advanced configuration option. Denotes the period for which model snapshots
```

Comment:
@szabosteve can you look over these docs changes please. Do they make sense?

Reply:
@davidkyle Sorry, I haven't noticed this one earlier.